.gitignore

```diff
@@ -84,7 +93,8 @@ erl_crash.dump
 .nfs*
 
 ### macOS ###
-*.DS_Store
+# General
```

Comment:
Is it properly nested? :p 
<img width="117" alt="image" src="https://user-images.githubusercontent.com/10851084/203531210-b04749bd-7848-4aad-a051-ea07ef25b999.png">